network/p2p/gossip/bloom_test.go

```diff
@@ -16,14 +16,14 @@ import (
 
 func TestBloomFilterRefresh(t *testing.T) {
 	tests := []struct {
-		name                     string
-		falsePositiveProbability float64
-		add                      []*testTx
-		expected                 []*testTx
+		name                          string
+		resetFalsePositiveProbability float64
```

Comment:
I feel like we should either keep the prior name for this var or introduce the target var. if feels weird to call it the reset probability and then use it for both the target and the reset values

Reply:
This test was trash, so I just redid it.